lib/models/challenge.dart

```diff
@@ -0,0 +1,12 @@
+import 'difficulty.dart';
```

Comment:
Let's avoid using relative imports for files in lib/.

Use the absolute import by adding the package name.

I think for this case, the absolute import would be similar to this (double check for errors in your IDE):
`import 'package:anxiety_go/models/difficulty.dart';`

Reply:
_"Let's avoid using relative imports for files in lib/."_

Any particular reason for this? Is this common practice to do so?

Reply:
Yes. This is a common practice and a futter convention. Let's keep it consistent throughout the project.

Reply:
Switched to absolute path.